recipe/meta.yaml

```diff
@@ -1,27 +1,28 @@
 {% set name = "dask" %}
-{% set version = "2022.2.1" %}
+{% set version = "2022.5.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b699da18d147da84c6c0be26d724dc1ec384960bf1f23c8db4f90740c9ac0a89
+  sha256: 0afd69dd0cd9f838fc0710eda1f3e3333d6603b37e93ded7fac4a51d77566a0f
 build:
   number: 0
-  noarch: python
+  skip: true  # [py<38]
 
 requirements:
   host:
     - python
   run:
     - python >=3.8
-    - bokeh >=2.1.1
+    - bokeh >=2.4.2
```

Comment:
I would add an `,<3.0` upper bound here, as Bokeh 3.0 will be ABI breaking. See AnacondaRecipes/repodata-hotfixes#155.

Reply:
added in new commit